### ISSUE TITLE
Dispatch to a Job from custom button callback

### ISSUE THREAD

**Issue by Elyahou** [sleeping-owl/admin#139]
Is it possible to add this possibility ?

**Comment by sleeping-owl** [comment]
In laravel 5.0 you can write something like this in your callback closure:
```php
\Bus::dispatch(new MyCommand(…));
```

I don't know is it works in 5.1 but I think there must be something like that.

**Comment by nunomazer** [comment]
Yes, I'm using Laravel 5.1 and doing `\Bus::dispatch(new MyCommand(…));` in several points of the code
